espresso-notes.md

```diff
@@ -0,0 +1,23 @@
+## Sequencer API
+
+### Status API
+
+```shell
+curl -fsSL http://localhost:8770/status/block-height
```

Comment:
I guess we can have a note, but going through the load balancer, right?

Reply:
yeah, it's all WIP I was just testing which endpoints works